src/ProductTour/Checkpoint.scss

```diff
@@ -89,12 +89,19 @@ $checkpoint-arrow-transparent: solid $checkpoint-arrow-width transparent;
   .pgn__checkpoint-header {
     display: flex;
     justify-content: space-between;
-    margin-bottom: map.get($spacers, 2\.5);
+    margin-bottom: map_get($spacers, 2);
+    align-items: center;
+    height: 30px;
```

Comment:
[question]: I don't have access to the design, but it seems to me that we shouldn't hardcode set the height for `pgn__checkpoint-header`. Maybe we should resize the Close button (which is currently stretching this block), or even remove the height altogether. These are just my thoughts. What do you think?